packages/datadog-plugin-mysql2/src/index.js

```diff
@@ -57,11 +57,19 @@ function wrapExecute (tracer, config, execute) {
 }
 
 function wrapCallback (tracer, span, parent, done) {
-  return tracer.scope().bind((error, res) => {
-    span.addTags({ error })
+  return tracer.scope().bind((...args) => {
+    const [ err ] = args;
+    if (err) {
```

Comment:
No need to expand to this condition, the previous code can stay as is as it is simpler.

Reply:
@rochdev sounds good. I was just trying to conform to how it was in mysql plugin but ill update!